test/parallel/test-http-flush-response-headers.js

```diff
@@ -0,0 +1,28 @@
+'use strict';
+const common = require('../common');
+const assert = require('assert');
+const http = require('http');
+
+const server = http.createServer();
+
+server.on('request', function(req, res) {
+  res.writeHead(200, {'foo': 'bar'});
+  res.flushHeaders();
+  res.flushHeaders();
```

Comment:
Saw another test doing this. `flushHeaders` should be idempotent, this validates that nothing fatal happens if it is called twice